### PR TITLE
fix(loading-message): display message with overlay

### LINKED ISSUE #444: Bug: Loading message does not render message without an overlay?
**Expected behavior -**

The `LoadingMessage` component should always render the message.

**Actual behavior -**

I _think_ the message is not rendered when the component does not have an overlay. See conversation: https://github.com/carbon-design-system/ibm-security/pull/369#discussion_r389143035

### LINKED ISSUE #447: `LoadingMessage` missing storybook knobs and props documentation
## Bug - `LoadingMessage` missing storybook knobs and props documentation

**Expected behavior -**

The `LoadingMessage` storybook demo should have functional knobs and also props documentation.

**Actual behavior -**

The `LoadingMessage` storybook demo is missing these features: https://ibm-security.carbondesignsystem.com/?path=/story/components-loading--with-message

Only some props are documented in the "Show info" panel.

### Steps for reproducing

See: https://ibm-security.carbondesignsystem.com/?path=/story/components-loading--with-message

### PR DESCRIPTION
**Pull request by SimonFinney**
## Affected issues

- Resolves #444 
- Resolves #447 

## Proposed changes

- Fix issue where `LoadingMessage` was not being included with the `withOverlay` prop, add styles to accommodate this, and refactor the logic
- Fix issue with story knobs that weren't rendering, including message, and refactor stories
- Update prop documentation to better reflect language elsewhere

### CODE REVIEW

File: src/components/Loading/LoadingMessage/LoadingMessage.js [hunk 1]
```diff
@@ -15,51 +23,41 @@ const LoadingMessage = ({
...
-}) => {
-  const loading = (
+}) => (
+  <div
+    className={classnames(getComponentNamespace('loading-message'), className, {
+      [overlayNamespace]: withOverlay,
+      [`${overlayNamespace}--stop`]: withOverlay && !active,
+    })}
+  >
     <Loading active={active} small={small} withOverlay={false} {...other} />
```
Comment (jendowns) [inline src/components/Loading/LoadingMessage/LoadingMessage.js:33]: Did you notice that `withOverlay` is set to `false` here? 🤔 Was wondering why that is...?

Comment (SimonFinney) [reply src/components/Loading/LoadingMessage/LoadingMessage.js:33]: Sorry, I'm only seeing this now! I'm not entirely sure - I think @j1mie implemented this initially, but I believe it's to do with being able to position the message next to the `Loading` component correctly

**Pull request merged**